Sources/Testing/Running/Runner.Context.swift

```diff
@@ -0,0 +1,204 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+extension Runner {
+  /// A type which collects the task-scoped context for a running ``Runner``
+  /// instance, the tests it runs, and other objects it interacts with.
+  ///
+  /// This type is intended for use via the task-local
+  /// ``Runner/_Context/current`` property.
+  fileprivate struct _Context: Sendable {
```

Comment:
The style we've adopted is to _not_ put an underscore before `fileprivate` symbols, just `private` ones.